web/controllers/scheduled_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule VerkWeb.ScheduledController do
+  use VerkWeb.Web, :controller
+
+  @schedule_key "schedule"
+
+  def index(conn, params) do
+    paginator = VerkWeb.RangePaginator.new(Verk.SortedSet.count!(@schedule_key, Verk.Redis), params["page"], params["per_page"])
```

Comment:
I will create an issue to have a `ScheduledSet` so that we don't need to use SortedSet directly.